markdown/org/docs/patterns/waralee/cutting/en.md

```diff
@@ -1,18 +1,37 @@
-Waralee consists of two identical pieces of fabric.
+<Note>
+
+Waralee pants parts consists of two mirrored pieces of fabric.
 They are basically big rectangles with pieces missing.
-As such, it is easier to print out the piece you need to cut out and measure the rest,
+As such, it is easier to print out the part you need to cut out and measure the rest,
 than it is to print out everything and cut the pattern the traditional way.
 
-The pattern is available both ways.
-You can print out the whole thing, tape all the pages together, and use that to cut out the fabric.
-It is also possible to print out only the part that you need to cut out.
-This part has a negative seam allowance, since you'll be using it opposite of what you normally use the pattern pieces for.
+However the pattern is available both ways so you can print out the whole thing, tape all the pages together, and use that to cut out the fabric if you prefer but we recommend only printing the part that you need to cut out.
```

Comment:
This sentence felt a little long to me and I had to reread it. A small suggestion:
```suggestion
That said, the pattern is available both ways. So you can print out the whole thing, tape all the pages together, and use that to cut out the fabric, if you prefer. But we recommend only printing the part that you need to cut out.
```

Reply:
yea this better paced. I mostly just did basica reformat not wording work on this updated